exercises/practice/allergies/.approaches/bitwise-and-case/snippet.txt

```diff
@@ -0,0 +1,4 @@
+toList : Int -> List Allergy
```

Comment:
I tried this, I don't think there is any shortening that sticks within the snippet limit and that also makes any sense. I agree though it would be nice to include it.

Reply:
>     A maximum of 8 lines can be used

I had not realized that, yeah, definitely not possible :(